app/Repositories/Contracts/NotificationRepository.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App\Repositories\Contracts;
+
+use App\Eloquent\Notification;
+
+interface NotificationRepository extends AbstractRepository
+{
+	//
+}
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Generic.WhiteSpace.DisallowTabIndent.TabsUsed</br>**Severity**: error</br>**File**: app/Repositories/Contracts/NotificationRepository.php</br>Spaces must be used to indent lines; tabs are not allowed